chunky/src/java/se/llbit/chunky/world/region/MCRegion.java

```diff
@@ -263,7 +299,7 @@ private static ChunkDataSource readChunkData(RandomAccessFile file, ChunkPositio
     int locationEntry = file.readInt();
     int sectorCount = locationEntry & 0xFF;
     int sectorOffset = locationEntry >> 8;
-    if (sectorOffset == 0 && sectorCount == 0) {
+    if (sectorOffset == 0 || sectorCount == 0) {
```

Comment:
This change doesn't seem right. Is it bad if the sectorOffset is 0 but the sectorCount is not zero?
```suggestion
    if (sectorOffset == 0 && sectorCount == 0) {
```

Reply:
If either is zero, the header entry is invalid.
Was concerned that we would be running through the later code with an invalid header entry.

Reply:
Also now that I think about it, a `sectorOffset` of `1` is also invalid as that points to the timestamp sector.
We probably don't need to think about that though.